setup.py

```diff
@@ -48,7 +48,7 @@
     "test": ["pytest", "pytest-cov", "flaky"],
     "wheel": ["wheel", "check-wheel-contents"],
 }
-for key in ["style", "test", "wheel"]:
+for key in ["docs", "style", "test", "wheel"]:
     extras_require["dev"] += extras_require[key]
```

Comment:
I think we just forgot to add those, but makes sense to me to enable devs to also build docs locally. At least I often do that when their build fails and the online output is less than readable.

Reply:
agreed verbally